data/hails.txt

```diff
@@ -317,22 +327,122 @@ phrase "friendly civilian"
 		"at a live concert"
 		"playing a gig"
 		"live in concert"
+	word
+		" at Tarazed Stadium"
+		" at the amphitheater"
+		" at a concert hall"
+		" in a nightclub"
+		" at Betelgeuse Stadium"
+		" at Lionheart Arena"
+		" at the Grange"
+		" at a resort"
+		" at Canyonland Park"
+		" in a stylin' party-hall"
+		""
+	word
+		" on Skymoot!"
+		" on New Sydney!"
+		" on Splashdown!"
+		" on Hestia!"
+		" on Earth!"
+		" on Heartland!"
+		" on Vail!"
+		" on New China!"
+		"!"
+
+phrase "friendly civilian"
+	word
+		"You should check out the latest installment of "
+		"I gotta admit, I'm hooked on "
+		"Once I finish my trip, I'm catching up on "
+		"I'm off to a fan convention for "
+		"Gotta say, I enjoy watching "
+	phrase
+		"media name"
+	word
+		". It's "
+		", "
+	word
+		""
+		"definitely "
+		"easily "
+		"maybe "
+		"perhaps "
+	word
+		"the "
+	word
+		"best "
+		"coolest "
+		"weirdest "
+		"funniest "
+		"most shocking "
+		"most fascinating "
+		"most surreal "
+		"most surprising "
+		"most enjoyable "
+		"most original "
+		"most engrossing "
+	phrase
+		"visual media"
 	word
 		" "
 	word
-		"at Tarazed Stadium!"
-		"at the amphitheater on Skymoot!"
-		"at a concert hall in New Sydney!"
-		"in a nightclub on Splashdown!"
-		"at the Syndicated Systems Velodrome!"
-		"at Betelgeuse Stadium!"
-		"on Hestia!"
-		"on Earth!"
-		"at Lionheart Arena!"
-		"at the Grange on Heartland!"
-		"at a resort on Vail!"
-		"at Canyonland Park!"
-		"in a stylin' party-hall on New China!"
+		"out there!"
+		"I've seen in ages!"
+		"this year!"
+		"yet!"
+		"since Seinfeld."
```

Comment:
Might want to stay away from direct modern references like this.

Reply:
It was intended to be a joke that Seinfeld was still considered relevant in the far future, but I see how it could be distracting. I've removed it.